src/axom/quest/tests/quest_point_in_cell_mfem.cpp

```diff
@@ -74,10 +74,36 @@ enum MeshType
   C_SHAPED_MESH        // Single element highly curved quadratic mesh (Lagrange)
 };
 
+template <typename ExecSpace>
+struct ExecTraits
+{
+  static int getAllocatorId()
+  {
+#ifdef AXOM_USE_UMPIRE
+    return axom::getUmpireResourceAllocatorID(
+      umpire::resource::MemoryResourceType::Host);
+#else
+    return axom::getDefaultAllocatorID();
+#endif
+  }
+};
+
+#ifdef AXOM_USE_CUDA
+template <int BLK_SZ>
+struct ExecTraits<axom::CUDA_EXEC<BLK_SZ>>
+{
+  static int getAllocatorId()
+  {
+    return axom::getUmpireResourceAllocatorID(
+      umpire::resource::MemoryResourceType::Device);
+  }
+};
+#endif
```

Comment:
Minor: Is this something that should be built into axom? Or something we'd expect users to have to write?

Reply:
Normally, we could use `axom::execution_space<ExecSpace>::allocatorID()`, but since the default for that is unified memory for `ExecSpace == CUDA_EXEC`, I have it explicitly coded for device-only memory so we can catch improper memory accesses.

My impression of the user case is that if they aren't specifying an allocator ID in `PointInCell`, they're probably fine with our default memory space.